todoapp/app/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 
     buildTypes {
         debug {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
We made this mandatory for the samples, you might have to modify the proguard files.